examples/MidiKeyboard/MidiKeyboardExampleUI.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * DISTRHO Plugin Framework (DPF)
+ * Copyright (C) 2012-2019 Filipe Coelho <falktx@falktx.com>
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any purpose with
+ * or without fee is hereby granted, provided that the above copyright notice and this
+ * permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD
+ * TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN
+ * NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL
+ * DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER
+ * IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+ * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#include "DistrhoUI.hpp"
+#include "Window.hpp"
+#include "KeyboardWidget.hpp"
+
+START_NAMESPACE_DISTRHO
+
+
+// -----------------------------------------------------------------------------------------------------------
+
+class MidiKeyboardExampleUI : public UI,
+                              public KeyboardWidget::Callback
+{
+  public:
+    /* constructor */
+    MidiKeyboardExampleUI()
+        : UI(800, 132),
```

Comment:
I would put the initial width and height as static vars, and then use them in the constructor.
This way we avoid calling `getWidth`, as we already know what it is anyway